__tests__/app.test.js

```diff
@@ -135,3 +135,108 @@ describe('/api/articles/:article_id/comments', () => {
 });
 
 
+describe('/api/articles/:article_id/comments', () => {
+    test('POST request reponds with 201 and the added comment', () => {
+        const testComment =   {
+            username: 'thebear22',
+            body: 'This is awesome!!!'
+          }
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send(testComment)
+        .expect(201)
+        .then(({ body }) => {
+            expect(body.comment).toMatchObject({
+                comment_id: expect.any(Number),
+                author: 'thebear22',
+                body: 'This is awesome!!!',
+                article_id: 1
+            })
+        })
+    });
+
+    test('POST requests body must contain object with username AND body properties', () => {
+        const testComment =   {
+            username: 'thebear22'
+          }
+        return request(app)
+        .post('/api/articles/1/comments')
+        .send(testComment)
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe('missing object properties')
+        });
+    });
+
+
+    test('POST requests responds with 400 status and error message for an invalid id ', () => {
+        const testComment =   {
+            username: 'thebear22',
+            body: 'This is awesome!!!'
+          }
+        return request(app)
+        .post('/api/articles/woof/comments')
+        .send(testComment)
+        .expect(400)
+        .then(( { body }) => {
+            expect(body.msg).toBe('invalid id')
+        })
+    });
+
+
+    test('POST requests responds with 404 status and error message for a valid id that does not exist', () => {
+        const testComment =   {
+            username: 'thebear22',
+            body: 'This is awesome!!!'
+          }
+        return request(app)
+        .post('/api/articles/1234/comments')
+        .send(testComment)
+        .expect(404)
+        .then(( { body }) => {
+            expect(body.msg).toBe('article id does not exist')
+        })
+    });
+});
+
+describe('/api/articles/:article_id', () => {
+    test('PATCH request responds with 201 with updated article votes', () => {
+        const testUpdate = { inc_votes: 1 }
+        return request(app)
+        .patch('/api/articles/1')
+        .send(testUpdate)
+        .expect(201)
```

Comment:
Would say a 200 or 202 more appropriate here - 201 for "created" a brand new resource